src/layout/ErrorLayout/ErrorLayout.module.scss

```diff
@@ -2,12 +2,12 @@
 @import 'styles/govBanner';
 @import 'styles/sfds/sfdsDependencies';
 
-$hero-image: url('/assets/images/hero.jpeg');
+$hero-image: url('/assets/images/hero-transparent.png');
 
 .errorContainer {
   display: flex;
   flex-direction: column;
-  min-height: 100vh;
```

Comment:
Was removing this intentional? On a larger screen I'm seeing a big gap at the bottom of the page.
![Screenshot 2023-04-10 at 11 25 55 AM](https://user-images.githubusercontent.com/99674188/230945951-05dddca3-31e5-4150-9963-501dd578aa94.png)

Reply:
oop i guess it was doing something after all, good catch lol. im going to put that back